### ISSUE TITLE
Markdown supported in swagger-editor but not swagger-codegen

### ISSUE THREAD

**Issue by joe-kahuna** [swagger-api/swagger-codegen#1203]
I can use Markdown in several places in my API spec (in YAML) to enter detailed descriptions, and swagger-editor will interpret it and display it correctly. However, when I put the resulting spec (in JSON) into swagger-codegen, it doesn't convert the markdown while generating a static HTML page. As far as I can tell, swagger-codegen doesn't accept YAML, so the end result is that I have to do extensive reformatting to convert my spec to HTML documentation. This has to be fixed.

**Comment by wing328** [comment]
@joe-kahuna you're correct in saying that the codegen won't do anything with markdown when generating the static documentation and we would definitely welcome your contribution with a PR to support that.

You might also want to take a look at https://github.com/Swagger2Markup/swagger2markup to see if it meets your requirement.

**Comment by joe-kahuna** [comment]
I'm not going to get to a markdown converter within codegen anytime soon.
It's low priority on my plate, simply because I'm not highly dependent on
swagger at the moment. swagger2markup would work for me, and I'll explore
it.

Joe

On Tue, Sep 8, 2015 at 8:55 PM, wing328 <notifications@github.com> wrote:

> @joe-kahuna <https://github.com/joe-kahuna> you're correct in saying that
> the codegen won't do anything with markdown when generating the static
> documentation and we would definitely welcome your contribution with a PR
> to support that.
>
> You might also want to take a look at
> https://github.com/Swagger2Markup/swagger2markup to see if it meets your
> requirement.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swagger-api/swagger-codegen/issues/1203#issuecomment-138771900>
> .
>



-- 
Joe Malin
Senior Technical Writer
Kahuna, Inc.

**Comment by wing328** [comment]
Understood and glad to hear that swagger2markup works for you.
